### PR TITLE
Moghes Immunities: Hexaisin Edition

### PR DESCRIPTION
**Pull request by LordPapalus**
## About The Pull Request

Changes how Moghes species are affected by Capsaisin, Condensed Capsaisin and adds a new chemical: Hexaisin.

Hexaisin is a chemical halfway between Capsaisin and Condensed Capsaisin. Currently only present in Moghes Jerky.

Moghes Foods are moved to Hospitality instead of being a Contraband. C'mon, NT has contracts both with Xsyssorr and OSS.

Changes how non-Moghes species are affected by Moghes foods and Hexaisin

Changes how Naramadi are affected by lactose.

## Why It's Good For The Game

Nothing I do is

## Changelog

:cl:
add: Hexaisin, new variant of Capsaisin. Present in Moghes foods only.
add: Lactose intolerance for Naramadi. Affects everything that logically has lactose.
/:cl:

### CODE REVIEW

File: code/modules/species/station/standard/unathidigi.dm
Comment (TheLordME) [inline code/modules/species/station/standard/unathidigi.dm:1]: this should be inherited

Comment (LordPapalus) [reply code/modules/species/station/standard/unathidigi.dm:1]: Logically, yes, but I don't have the time or gummy bears to bear testing it. I'll comment it out and see how it functions in game. Bug reports are easy enough to make tbh.

**Comment by LordPapalus** [comment]
Now adds species-wide lactose intolerance for Naramadi. Tested, compiled, worked.

File: code/modules/reagents/chemistry/reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 1]
```diff
@@ -4539,6 +4608,16 @@
 	color = "#EDB91F"
 	taste_description = "cheese"
 
+/datum/reagent/nutriment/protein/cheese/affect_ingest(mob/living/carbon/M, alien, removed) //Cheese is a kind of milk.
```
Comment (TheLordME) [inline code/modules/reagents/chemistry/reagents/Chemistry-Reagents-Food-Drinks.dm:4611]: the irony of cheese heads being unable to eat cheese

**Review by TheLordME — APPROVED** [review]

**Pull request merged**